VAMobile/ios/VAMobile/AppDelegate.mm

```diff
@@ -58,16 +57,6 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   return YES;
 }
 
-/// This method controls whether the `concurrentRoot`feature of React18 is turned on or off.
-///
-/// @see: https://reactjs.org/blog/2022/03/29/react-v18.html
-/// @note: This requires to be rendering on Fabric (i.e. on the New Architecture).
-/// @return: `true` if the `concurrentRoot` feature is enabled. Otherwise, it returns `false`.
-- (BOOL)concurrentRootEnabled
-{
-  // Switch this bool to turn on and off the concurrent root
-  return false;
```

Comment:
Should this have been removed? I see it was in the upgrade helper, but also we had it disabled and [searching on it the doc indicates it's an opt out](https://reactnative.dev/docs/react-18-and-react-native#opt-ing-out-of-react-18-on-ios). Not sure if it's trying to run with it on without it being explicitly false and that could be causing issues.